<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Windows_Main</class>
 <widget class="QMainWindow" name="Windows_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMTAphyViewer 0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>9</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionCloseDataSource"/>
    <addaction name="separator"/>
    <addaction name="actionView_Scenario"/>
    <addaction name="actionView_CouchDB_Trace"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAboutWrowser">
   <property name="text">
    <string>About Wrowser</string>
   </property>
  </action>
  <action name="actionOpenDatabase">
   <property name="text">
    <string>Open S&amp;QLObject Database</string>
   </property>
  </action>
  <action name="actionOpenDSV">
   <property name="text">
    <string>Open DS&amp;V</string>
   </property>
   <property name="iconText">
    <string>Open DSV</string>
   </property>
  </action>
  <action name="actionOpenPythonCampaign">
   <property name="text">
    <string>Open &amp;Python Campaign</string>
   </property>
  </action>
  <action name="actionOpenDirectory">
   <property name="text">
    <string>Open &amp;Directory</string>
   </property>
  </action>
  <action name="actionCloseFigure">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionCloseDataSource">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionNewLogEval">
   <property name="text">
    <string>LogEval</string>
   </property>
  </action>
  <action name="actionNewTimeSeries">
   <property name="text">
    <string>TimeSeries</string>
   </property>
  </action>
  <action name="actionNewLRE">
   <property name="text">
    <string>(D)LRE</string>
   </property>
  </action>
  <action name="actionNewBatchMeans">
   <property name="text">
    <string>BatchMeans</string>
   </property>
  </action>
  <action name="actionNewTable">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="actionNewXDF">
   <property name="text">
    <string>PDF/CDF/CCDF</string>
   </property>
  </action>
  <action name="actionNewParameter">
   <property name="text">
    <string>Parameter</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>C&amp;onfigure...</string>
   </property>
  </action>
  <action name="actionOpenCampaignDatabase">
   <property name="text">
    <string>Open &amp;Campaign Database</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
  </action>
  <action name="actionView_Scenario">
   <property name="text">
    <string>View &amp;Scenario</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionView_CouchDB_Trace">
   <property name="text">
    <string>View CouchDB Trace</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="viewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>Windows_Main</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
